yaml-tests/src/test/resources/disabled-planner-rewrites/uuid-proto.yamsql

```diff
@@ -0,0 +1,96 @@
+#
+# uuid-proto.yamsql
+#
+# This source file is part of the FoundationDB open source project
+#
+# Copyright 2021-2025 Apple Inc. and the FoundationDB project authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+setup:
+  connect: "jdbc:embed:/__SYS?schema=CATALOG"
+  steps:
+    - query: drop schema template if exists uuid_proto_template
+    - query: drop database if exists /FRL/UUID_PROTO_YAML
+    - query: create database /FRL/UUID_PROTO_YAML
```

Comment:
In theory, we should pick a different database path for this test. There are a couple of others (mostly the `-proto` and `-metadata` tests, I think) that hard-code their database URI. I think it only actually matters if we try and parallelize running two test files at the same time, which we currently don't do though we've wanted to at certain points, as they need different paths to ensure isolation

Reply:
As far as I can tell this is the same problem when running `YamlIntegrationTests`. I'll leave this for another time.